components/css/global/_base.scss

```diff
@@ -1,163 +1,124 @@
-	// padding-bottom:0.25em;
-	margin-top:0;
-	margin-bottom:0.25em;
-	font-family:$font-champion-welter;
-	color:$color-headlineGold;
-    font-size: calc-toEm(36);
-    line-height:1.2;
-	text-transform: uppercase;
-}
-h2{
-	//padding-top:1em;
-	//padding-bottom:0.15em;
-	margin-top:1rem;
-	margin-bottom:0.15em;
-	font-family:$font-archivo-narrow;
-	color:$color-headlineGold;
-	font-size: calc-toEm(32);
-	line-height:1.2;
-}
-h3{
-	//padding-top:1em;
-	margin-top:1rem;
-	font-family:$font-archivo-narrow;
-	color:$color-gray70;
-	font-size: calc-toEm(28);
-	line-height:1.2;
-	font-weight:bold;
-}
-h4{
-	//padding-top:1em;
-	margin-top:1rem;
-	font-family:$font-archivo-narrow;
-	color:$color-headlineGold;
-	font-weight:bold;
-	font-size: calc-toEm(24);
-	line-height:1.2;
-	text-transform:uppercase;
-}
-h5{
-	//padding-top:1.5em;
-	margin-top:1rem;
-	font-family:$font-archivo-narrow;
-	font-weight:bold;
-	font-size: calc-toEm(22);
-	line-height:1.2;
-	text-transform:uppercase;
-}
-h6{
-	//padding-top:2em;
-	margin-top:1rem;
-	font-family:$font-archivo-narrow;
-	font-weight:bold;
-	font-size: calc-toEm(18);
-	line-height:1.2;
-	text-transform:uppercase;
-}
-h1,h2,h3,h4,h5,h6{
-	a:link, a:visited, a:hover, a:active {
-		color:inherit;
-		font-weight:inherit;
-	}
-}
-// h1,h2,h3,h4,h5,h6{
-// 	margin-top:0;
-// 	margin-bottom:0;
-// }
-// h1+h3{
-// 	padding-top:0;
-// }
-// h3+h6,h4+h6{
-// 	padding-top:1em;
-// }
-// h4+h5,h5+h6{
-// 	padding-top:0.75em;
-// }
-// h2+h6,h3+h5,h3+h4{
-// 	padding-top:0.5em;
-// }
-// h1+h4,h1+h5,h1+h6,h2+h3,h2+h4,h2+h5{
-// 	padding-top:0.25em;
-// }
-// h1+h2{
-// 	padding-top:0.15em;
-// }
-p{
-	margin:0 !important;
-	padding:7.5px 0;
-}
-small{
-	font-size:0.75em;
-	line-height:1em;
-}
-big{
-	font-size:1.25em;
-	line-height:1em;
-}
-a:link, a:visited, a:active, a:hover{
-	color:#000;
-	font-weight:700;
-	text-decoration:underline;
-}
-a:hover {
-	text-decoration:none;
-}
-ul,ol{
-	padding:10px 0 10px 35px;
-}
-ul{
-	list-style-type:disc;
-}
-ul ul{
-	padding-top:0;
-	padding-bottom:0;
-	list-style-type:circle;
-}
-ul ul ul{
-	list-style-type:square;
-}
-strong,b{
-	font-weight:700;
-}
-i,em{
-	font-style:italic;
-}
-u{
-	text-decoration:underline;
-}
-.right {
-	float:right;
-}
-.left {
-	float:left;
-}
-.transition {
-	-webkit-transition-duration: .3s;
-	-moz-transition-duration: .3s;
-	-o-transition-duration: .3s;
-	transition-duration: .3s;
-	-ms-transition-duration: .3s;
-}
+html {
+	background-color: #3b3b3b;
+	color: #000;
+	font-size: 100%;
+}
+
+body {
+	font-family: $font-arial;
+}
+
+@media (min-width:768px) {
+	body {
+		font-size: 1em;
+		line-height: calc-toEm(24);
+	}
+}
+@media (max-width:767px) {
+	body {
+		font-size: calc-toEm(18);
+		line-height: calc-toEm(24);
+	}
+}
+
+h1 {
+	@include h1;
+}
+
+h2 {
+	@include h2;
+}
+
+h3 {
+	@include h3;
+}
+
+h4 {
+	@include h4;
+}
+
+h5 {
+	@include h5;
+}
+
+h6 {
+	@include h6;
+}
+
+
+h1, h2, h3, h4, h5, h6 {
+	a:active, a:hover, a:link, a:visited {
```

Comment:
Although it doesn't matter in this case, lets make sure the order is always:
`a:link, a:visited, a:hover, a:active`

Reply:
Done